hwilib/devices/trezorlib/btc.py

```diff
@@ -261,7 +261,7 @@ def sign_tx(
         prev_txes = {}
 
     if details is not None:
-        warnings.warning(
+        warnings.warn(
```

Comment:
In a5a68961b3bd625c4ab6849f58003a7707d7b55c "restored"

What's the purpose of this commit?

Reply:
It should be logging.warning as logging.warn is deprecated, but for warnings it is warnings.warn.
So I restored the correct one. Please squash and merge as the individual commits have no value